sympy/parsing/tests/test_sympy_parser.py

```diff
@@ -193,6 +190,11 @@ def test_recursive_evaluate_false_10560():
         assert parse_expr(text, evaluate=False) == parse_expr(result, evaluate=False)
 
 
+def test_issue__evaluateFalse_24288():
+    for u in [op(1, 2, evaluate=False) for op in [Eq, Ne, Lt, Le, Gt, Ge]]:
```

Comment:
I am getting an Assertion Error in that but when I run in my editor that run perfectly.
How should I fix

Reply:
I just tested the code in this PR and I get:
```
In [12]: parse_expr('1 < 2', evaluate=False)
Out[12]: True
```

Reply:
It shouldn't give True, it should give `1 < 2`.

Reply:
> It shouldn't give True, it should give `1 < 2`.

Now can I move forward with that code ?